datafusion/sql/src/unparser/expr.rs

```diff
@@ -711,6 +724,15 @@ mod tests {
                 Expr::IsUnknown(Box::new((col("a") + col("b")).gt(lit(4)))),
                 r#"(("a" + "b") > 4) IS UNKNOWN"#,
             ),
+            (
+                Expr::Between(Between {
```

Comment:
I think you could write this more concisely using https://docs.rs/datafusion/latest/datafusion/logical_expr/enum.Expr.html#method.between

so like `betwen(col("a:"), lit(1), lit(7))`